src/main/java/edu/cornell/kfs/coa/batch/service/impl/CreateClosedLaborAccountsCsvByPreviousDaysServiceImpl.java

```diff
@@ -0,0 +1,226 @@
+            writeClosedLaborAccountsToCsvFormattedFile(fileDataRangeSpecified);
+        } else {
+            LOG.error("createClosedLaborAccountCsvByParameterPastDays: NO FILE CREATED. REQUIRED PARAMETER IS NOT SET.");
+        }
+    }
+    
+    private Map<String, Integer> parameterValueForTypeOfClosedAccountFileToCreate() {
+        Map<String, Integer> fileDataContentType = new HashMap<String, Integer>();
+        Integer numberOfDaysPrevious = getLaborAccoutsClosedPastDaysParameter();
+        
+        if (numberOfDaysPrevious.intValue() == CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.PARAMETER_LABOR_ACCOUNTS_CLOSED_OVER_PAST_DAYS_NOT_SET) {
+            fileDataContentType.put(CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FILE_DATA_CONTENT_TYPE_IS, CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FILE_DATA_CONTENT_TYPES.NO_PARMETER_FOUND);
+            LOG.info("parameterValueForTypeOfClosedAccountFileToCreate: Detected parameter LABOR_ACCOUNTS_CLOSED_OVER_PAST_DAYS required for batch job execution is not set.");
+            
+        } else if (numberOfDaysPrevious.intValue() == CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.PARAMETER_SET_TO_CREATE_FULL_SEED_FILE) {
+            fileDataContentType.put(CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FILE_DATA_CONTENT_TYPE_IS, CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FILE_DATA_CONTENT_TYPES.SEED);
+            LOG.info("parameterValueForTypeOfClosedAccountFileToCreate: Full seed file will be created.");
+            
+        } else {
+            fileDataContentType.put(CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FILE_DATA_CONTENT_TYPE_IS, CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FILE_DATA_CONTENT_TYPES.RANGE);
+            fileDataContentType.put(CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FROM_DATE, numberOfDaysPrevious.intValue());
+            LOG.info("parameterValueForTypeOfClosedAccountFileToCreate: File will contain data foe the previous " + numberOfDaysPrevious.intValue() +  " days.");
+        }
+        
+        return fileDataContentType;
+    }
+    
+    private Integer getLaborAccoutsClosedPastDaysParameter() {
+        String daysPreviousParameterValue = getParameterService().getParameterValueAsString(KFSConstants.CoreModuleNamespaces.CHART, 
+                CUKFSParameterKeyConstants.ALL_COMPONENTS, CuCoaBatchParameterConstants.LABOR_ACCOUNTS_CLOSED_OVER_PAST_DAYS);
+        
+        int daysPreviousInt;
+        try {
+            daysPreviousInt = Integer.parseInt(daysPreviousParameterValue);
+        } catch (NumberFormatException ne) {
+            LOG.error("getLaborAccoutsClosedPastDaysParameter: Namespace KFS-COA system parameter LABOR_ACCOUNTS_CLOSED_OVER_PAST_DAYS is either not set or could not be converted to a number.");
```

Comment:
I would put the value of daysPreviousParameterValue in the log message

Reply:
Done.